packages/dds/tree/src/tree/treeTextFormat.ts

```diff
@@ -168,14 +169,33 @@ export function genericTreeKeys<T>(tree: GenericFieldsNode<T>): readonly FieldKe
         if (global === undefined) {
             return [];
         }
-        return (Object.getOwnPropertyNames(global) as GlobalFieldKey[]).map(symbolFromKey);
+        return (Object.getOwnPropertyNames(global) as GlobalFieldKey[])
+            .filter((key: GlobalFieldKey) => {
+                const value: Value = global[key];
+                return !Array.isArray(value) || value.length !== 0;
```

Comment:
If we do want to keep this filtering approach, there is no reason to call `Array.isArray` here. The data has to be an array due to the types and checking this adds unneeded overhead.

Reply:
Why does this have to be an array? Is it a property of global fields?

Reply:
Oh I see, cause FieldMapObject only contains arrays as values.

Reply:
JsonableTree stores all fields as arrays (and the global field map only stores fields). The value "Value" is only used for values on nodes (which are not fields)